ln-gateway/src/webserver.rs

```diff
@@ -33,7 +34,8 @@ pub async fn pay_invoice(
 pub async fn run_webserver(sender: mpsc::Sender<GatewayRequest>) -> axum::response::Result<()> {
     let app = Router::new()
         .route("/pay_invoice", post(pay_invoice))
-        .layer(Extension(sender));
+        .layer(Extension(sender))
+        .layer(CorsLayer::permissive());
```

Comment:
Intuitively this should be safe because there's no session or anything, it's an unauthenticated API endpoint. Is that reasoning correct? I'm not too familiar with this web stuff.

Reply:
Yes, the reasoning is correct. But [`permissive` config](https://docs.rs/tower-http/latest/tower_http/cors/struct.CorsLayer.html#method.permissive) doesn't allow `credentials` so the cookies, etc won't be sent.